OpenChange/iCalEvent+MAPIStore.m

```diff
@@ -457,7 +460,14 @@ - (void) updateFromMAPIProperties: (NSDictionary *) properties
                     {
                       [person setParticipationStatus: newPartStat];
                       [person setRsvp: @"TRUE"];
-                      [person setRole: @"REQ-PARTICIPANT"];
+                      if (i < ccFirstIndex)
```

Comment:
No, I don't know of any specific section explaining this (you are welcome to search for it if you think it's really important), I've just confirmed the behavior empirically with Outlook. Anyway, it's something basic and makes a lot of sense, don't know what kind of "details" you want...